pkg/cmd/alias/set/set.go

```diff
@@ -43,21 +43,28 @@ func NewCmdSet(f *cmdutil.Factory, runF func(*SetOptions) error) *cobra.Command
 			for commands with mixed quotes or multiple lines.
 
 			If '--shell' is specified, the alias will be run through a shell interpreter (sh). This allows you
-			to compose commands with "|" or redirect with ">". Note that extra arguments following the alias
+			to compose commands with '|' or redirect with '>'. Note that extra arguments following the alias
 			will not be automatically passed to the expanded expression. To have a shell alias receive
-			arguments, you must explicitly accept them using "$1", "$2", etc., or "$@" to accept all of them.
-
-			Platform note: on Windows, shell aliases are executed via "sh" as installed by Git For Windows. If
-			you have installed git on Windows in some other way, shell aliases may not work for you.
+			arguments, you must explicitly accept them using '$1', '$2', etc., or '$@' to accept all of them.
```

Comment:
Why changing this to single quotes?

Reply:
Ditto from the previous comment.  You'll notice the double quotes are used here, but in the first paragraph, single quotes are used to refer to the same variables.